src/panels/SkyBrowserPanel/hooks.ts

```diff
@@ -0,0 +1,223 @@
+import { useEffect } from 'react';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+import { useAppDispatch, useAppSelector } from '@/redux/hooks';
+import {
+  subscribeToSkyBrowser,
+  unsubscribeToSkyBrowser
+} from '@/redux/skybrowser/skybrowserMiddleware';
+import {
+  setActiveImage,
+  setImageCollectionData
+} from '@/redux/skybrowser/skybrowserSlice';
+
+export function useGetWwtImageCollection() {
+  const luaApi = useOpenSpaceApi();
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    async function getWwtListOfImages() {
+      if (!luaApi || !luaApi.skybrowser) {
+        return;
+      }
+      const imgData = await luaApi.skybrowser.listOfImages();
+      if (imgData) {
+        const images = Object.values(imgData);
+        if (images.length !== 0) {
+          const imgDataWithKey = images.map((image) => ({
+            ...image,
+            key: image.identifier
+          }));
+          dispatch(setImageCollectionData(imgDataWithKey));
+        }
+      } else {
+        throw new Error('No AAS WorldWide Telescope images!');
+      }
+    }
+    try {
+      getWwtListOfImages();
+    } catch (e) {
+      throw Error(`Could not load image collection from OpenSpace: ${e}`);
+    }
+  }, [luaApi, dispatch]);
+}
+
+export function useGetSkyBrowserData() {
+  useGetWwtImageCollection();
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    dispatch(subscribeToSkyBrowser());
+    return () => {
+      dispatch(unsubscribeToSkyBrowser());
+    };
+  }, [dispatch]);
+}
+
+export function useActiveImage(): [string, (url: string) => void] {
```

Comment:
What's the difference between the active and selected image?

Reply:
Oh, maybe the selected are the ones that I think of as "added" ?

Reply:
Yes - the active one is the one with the border. Selected / added are the ones in the list